modin/pandas/series.py

```diff
@@ -517,12 +520,19 @@ def f(x):
             with np.errstate(all="ignore"):
                 if isinstance(f, np.ufunc):
                     return f(self)
-                query_compiler = self.map(f)._query_compiler
+                result = self.map(f)._query_compiler
         if return_type not in ["DataFrame", "Series"]:
-            return query_compiler.to_pandas().squeeze()
+            # sometimes result can be not a query_compiler, but scalar (for example
+            # for sum or count functions)
+            from ..backends.base.query_compiler import BaseQueryCompiler
+
+            if isinstance(result, BaseQueryCompiler):
```

Comment:
Can we do `if isinstance(result, type(self._query_compiler)):` to avoid import?
```suggestion
            if isinstance(result, type(self._query_compiler)):
```

Reply:
Yes, your suggestion is better.